src/explore-education-statistics-frontend/src/services/tableBuilderService.ts

```diff
@@ -151,10 +148,10 @@ export default {
   ): Promise<PublicationSubjectMeta> {
     return dataApi.post('/meta/subject', query);
   },
-  getTableData(query: TableDataQuery): Promise<TableData> {
+  getTableData(query: TableDataQuery): Promise<FullTable> {
     return dataApi.post('/tablebuilder', query);
   },
-  getTablePermalink(query: TableDataQuery): Promise<TablePermalink> {
+  getTablePermalink(query: PermalinkCreate): Promise<TablePermalink> {
```

Comment:
- This should be in the `permalinkService` (and it's associated `TablePermalink`)
- Would rename it `createPermalink`, as it doesn't actually 'get' it.